hpx/runtime/serialization/container.hpp

```diff
@@ -31,7 +31,7 @@ namespace hpx { namespace serialization
             naming::gid_type const & split_gid) = 0;
         virtual void set_filter(binary_filter* filter) = 0;
         virtual void save_binary(void const* address, std::size_t count) = 0;
```

Comment:
Sorry. Misread what you were referring to. For save_binary, it always writes out the number of bytes supplied, so it didn't seem necessary to return the value - but if it makes the code look more consistent, then I'll add it.